cognite_toolkit/_cdf.py

```diff
@@ -248,7 +248,7 @@ def build(
 ) -> None:
     """Build configuration files from the module templates to a local build directory."""
     cmd = BuildCommand()
-    cmd.execute(ctx, Path(source_dir), Path(build_dir), build_env_name, no_clean)
+    cmd.execute(ctx.obj.verbose, Path(source_dir), Path(build_dir), build_env_name, no_clean)
```

Comment:
This is just a refactoring to make it easier to write tests for the `BuildCommand`. I used it to track down a bug triggered by a customer case. 

Note that the last implementation breaks the `inappropriate intimacy` principle by passing the entire `ctx` object in, when all that is need is the `verbose` flag.